From b76a9985a390db0eedf616dfec713deafd55e27c Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Wed, 18 Apr 2018 16:46:56 +0200
Subject: [PATCH 7/9] lineage: qcom: Enable media extensions for all QC devices

* This is now needed for the latest legacy HAL1 hacks, which were
  previously the only devices not using media extensions.
* Enable it for all QC devices to avoid running into
  issues with devices not having this enabled in the future.

Change-Id: I484840e712f7da6d0064a5f8016e8061b9cba838
---
 build/core/qcom_target.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build/core/qcom_target.mk b/build/core/qcom_target.mk
index 3e70b013..df1cd87b 100644
--- a/build/core/qcom_target.mk
+++ b/build/core/qcom_target.mk
@@ -45,6 +45,9 @@ ifeq ($(BOARD_USES_QCOM_HARDWARE),true)
         endif
     endif
 
+    # Enable media extensions
+    TARGET_USES_MEDIA_EXTENSIONS := true
+
     # Allow building audio encoders
     TARGET_USES_QCOM_MM_AUDIO := true
 
-- 
2.17.0

